### PR TITLE
worked on linter config

### PR DESCRIPTION
**Pull request by Mesi21**
- Add a new JavaScript file and import it as a module.
-Added methods related to the status updates (completed: true/false).
- Added event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
-Store the updated array of items in local storage

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Mesi21 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -1,67 +1,88 @@
...
+    localStorage.setItem('AllTasks', JSON.stringify(this.tasks));
+  }
 
+  toggleState(checkBox, item) {
+    if (checkBox.checked) {
+      this.item.completed = true;
+    } else {
+      this.item.completed = false;
+    }
+    return item;
```
Comment (mikenath223) [inline src/index.js:40]: - [x] After marking a task as completed and reloading the page the task completed status disappears both on the page and on local storage. Kindly fix it.

**Comment by mikenath223** [comment]
- [x] Presently the readme looks all clumped up. Kindly divide it into sections, matching this [template](https://github.com/microverseinc/readme-template)

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Mesi21 😃,

Well done working on the changes 👏 All seems fine to me 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**